schwifty/registry.py

```diff
@@ -11,8 +11,8 @@
 
 
 try:
-    from importlib.abc import Traversable
     from importlib.resources import files
+    from importlib.resources.abc import Traversable
 except ImportError:
     from importlib_resources import files  # type: ignore
     from importlib_resources.abc import Traversable  # type: ignore
```

Comment:
```suggestion
try:
    from importlib_resources import files
    from importlib_resources.abc import Traversable
except ImportError:
    from importlib.abc import Traversable  # type: ignore
    from importlib.resources import files  # type: ignore
```

Reply:
This would however not remove the `DeprecationWarning` that was complained about in #158, right?

Reply:
Maybe I was too quick indeed. What I wanted to say is that we should try to import the Python 3.12 way, then the Python 3.9/3.10 and finally 3.8

Reply:
For reference the error with 3.9 and 3.10 is:

```
ModuleNotFoundError: No module named 'importlib_resources'
```